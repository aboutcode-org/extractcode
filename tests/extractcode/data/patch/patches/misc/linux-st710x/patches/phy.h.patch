--- linux-2.6.23/include/linux/phy.h.orig	2009-02-04 16:33:03.000000000 +0100
+++ linux-2.6.23/include/linux/phy.h	2009-01-20 08:45:48.000000000 +0100
@@ -63,9 +63,10 @@
 
 
 #define PHY_INIT_TIMEOUT	100000
-#define PHY_STATE_TIME		1
-#define PHY_FORCE_TIMEOUT	10
-#define PHY_AN_TIMEOUT		10
+#define PHY_INIT_TIME           (HZ>>3)
+#define PHY_STATE_TIME		HZ
+#define PHY_FORCE_TIMEOUT	(10<<3)
+#define PHY_AN_TIMEOUT		(10<<3)
 
 #define PHY_MAX_ADDR	32
 
