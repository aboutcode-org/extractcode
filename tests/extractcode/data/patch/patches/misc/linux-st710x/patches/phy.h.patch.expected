[('linux-2.6.23/include/linux/phy.h.orig',
  'linux-2.6.23/include/linux/phy.h',
  '--- linux-2.6.23/include/linux/phy.h.orig\n+++ linux-2.6.23/include/linux/phy.h\n@@ -63,9 +63,10 @@\n\n\n#define PHY_INIT_TIMEOUT\t100000\n-#define PHY_STATE_TIME\t\t1\n-#define PHY_FORCE_TIMEOUT\t10\n-#define PHY_AN_TIMEOUT\t\t10\n+#define PHY_INIT_TIME           (HZ>>3)\n+#define PHY_STATE_TIME\t\tHZ\n+#define PHY_FORCE_TIMEOUT\t(10<<3)\n+#define PHY_AN_TIMEOUT\t\t(10<<3)\n\n#define PHY_MAX_ADDR\t32\n')]